src/main/scala/pl/project13/scala/akka/raft/Leader.scala

```diff
@@ -110,7 +110,7 @@ private[raft] trait Leader {
   }
 
   def replicateLog(m: LeaderMeta) {
-    m.membersExceptSelf foreach { member =>
+    m.membersExceptSelf filter(_ != self) foreach { member =>
       // todo remove me
```

Comment:
Yes, this fixes the particular bug I was interested in. There is only one other reference to membersExceptSelf in the Candidate trait when starting an election. Filtering out the self member seemed to have no effect on the election logic so I stuck to the single change in the place that was having the issue to avoid any unintended side effects in Candidate.